docs/test.ts

```diff
@@ -536,3 +536,100 @@ const otelTraceId: string = spanContext.traceId
 const otelSpanId: string = spanContext.spanId
 const otelTraceFlags: number = spanContext.traceFlags
 const otelTraceState: opentelemetry.TraceState = spanContext.traceState!
+
+// -- LLM Observability --
+const llmobsEnableOptions = {
+  apiKey: 'dd-api-key',
+  mlApp: 'mlApp',
+  agentlessEnabled: true
+}
+tracer.init({
+  llmobs: llmobsEnableOptions,
+  experimental: {
+    llmobs: llmobsEnableOptions
+  }
+})
+const llmobs = tracer.llmobs
+const enabled = llmobs.enabled
+
+// manually enable
+llmobs.enable({
+  apiKey: 'dd-api-key',
+  mlApp: 'mlApp',
+  agentlessEnabled: true
+})
+
+// manually disable
+llmobs.disable()
+
+// trace block of code
+llmobs.trace({ name: 'name', kind: 'llm' }, () => {})
```

Comment:
Is `name` still an optional argument? I recall we decided to make this required and move `kind` to be optional, but it's been a while so please correct me if I'm wrong 😓

Reply:
yeah we decided on it being required. i have the strict typing for that [here](https://github.com/DataDog/dd-trace-js/blob/7c7a328b2f52da0b9b515ca5d32e1af23bdf3097/index.d.ts#L2252), where `LLMObsNamedSpanOptions` is defined as an [llmobs span options with a required name argument](https://github.com/DataDog/dd-trace-js/blob/7c7a328b2f52da0b9b515ca5d32e1af23bdf3097/index.d.ts#L2500-L2505). I need to update the description of it to reinforce/state that it is required.

also, while I'm at that, I realized that I didn't really add any logic to enforce this in the actual `trace` implementation. Do we not want to send LLMObs spans if no name is provided in this case? Or just log a warning and give it an empty name. For TypeScript users, they actually won't be able to compile their applications if it's not given, which I understand a lot of users will be using typescript. But for those using plain CommonJS, they won't have that guardrail, and right now it's possible they could not specify name and it would create a span event with no name. wdyt?